jsonslice.go

```diff
@@ -1103,14 +1114,29 @@ func matchSubslice(str, needle []byte) bool {
 }
 
 func doFunc(input []byte, nod *tNode) ([]byte, error) {
-	var err error
-	var result int
-	if bytes.Equal(word("size"), nod.Keys[0]) {
-		result, err = skipValue(input, 0)
-	} else if bytes.Equal(word("length"), nod.Keys[0]) || bytes.Equal(word("count"), nod.Keys[0]) {
+	switch {
+	case bytes.Equal(word("size"), nod.Keys[0]):
+		res, err := skipValue(input, 0)
+		if err != nil {
```

Comment:
do we need this? can we just do
```
return []byte(strconv.Itoa(res)), err
```

seems like skipValue does a version of this if error conditional handling inside the method 
https://github.com/celene-isip-sp/jsonslice/blob/master/jsonslice.go#L1038

Reply:
> do we need this? can we just do
> 
> ```
> return []byte(strconv.Itoa(res)), err
> ```
> 
> seems like skipValue does a version of this if error conditional handling inside the method https://github.com/celene-isip-sp/jsonslice/blob/master/jsonslice.go#L1038

@celene-isip-sp well, yes, but I like to always check for errors, 🤷